### ISSUE TITLE
use dockerhub busybox image

### ISSUE THREAD

**Issue by js-9** [senthilrch/kube-fledged#181]
My install to an arm cluster went surprisingly well. The only thing that failed was the busybox docker image pull since gcr docker registry doesn't support multiarch images and it doesn't contain an arm image for busybox. Setting the helm variable to use just the "busybox" image fixed this issue, so I would propose to just use that.

**Comment by senthilrch** [comment]
@js-9 : Thanks for raising this issue. I understand the busybox image in gcr doesn't supports arm architecture. By default, kube-fledged doesn't uses the busybox image from dockerhub because the pulls are subject to ratelimiting.

I've copied the busybox:1.35.0 image from dockerhub to senthilrch/busybox:1.35.0 in dockerhub. The "senthilrch" namespace in dockerhub is exempted from ratelimiting. I've modified kube-fledged to use senthilrch/busybox:1.35.0 by default; this will fix the issue you faced with installing kube-fledged in arm cluster.
